proto/ServerSideExtension.proto

```diff
@@ -46,86 +43,104 @@ message Parameter {
+ * A number of rows collected in one message. The actual number will depend on the size of each row and is adjusted to optimize throughput.
  */
 message BundledRows {
     repeated Row rows = 1;
 }
 
 /**
- * A header sent at the start of an EvaluateScript request under the key "qlik-scriptrequestheader-bin"
+ * A header sent at the start of an EvaluateScript request under the key "qlik-scriptrequestheader-bin".
  */
 message ScriptRequestHeader {
-    string script = 1;              /// Script to be executed.
-    FunctionType functionType = 2;  /// Function type of the script evaluation, scalar, aggregation or tensor.
-    DataType returnType = 3;        /// Return type from the script evaluation, numeric, string or both.
-    repeated Parameter params = 4;  /// Parameters names and types passed to the script.
+    string script = 1;              /// The script to be executed.
+    FunctionType functionType = 2;  /// The function type of the script evaluation: scalar, aggregation or tensor.
+    DataType returnType = 3;        /// The return type from the script evaluation: numeric, string or both.
+    repeated Parameter params = 4;  /// The parameters names and types passed to the script.
 }
 
 /**
- * A header sent at the start of an ExecuteFunction request under the key "qlik-functionrequestheader-bin"
+ * A header sent at the start of an ExecuteFunction request under the key "qlik-functionrequestheader-bin".
  */
 message FunctionRequestHeader {
-    int32 functionId = 1;   /// Id of the function to be executed.
-    string version = 2;     /// Dummy variable as a workaround for an issue.
+    int32 functionId = 1;   /// The ID of the function to be executed.
+    string version = 2;     /// A dummy variable as a workaround for an issue.
 }
 
 /**
- * A header sent at the start of both an EvaluateScript request and an ExecuteFunction request under the key "qlik-commonrequestheader-bin"
+ * A header sent at the start of both an EvaluateScript request and an ExecuteFunction request under the key "qlik-commonrequestheader-bin".
  */
 message CommonRequestHeader {
-    string appId = 1;       /// Id of the app the request was executed in.
-    string userId = 2;      /// Id of the user the request was executed by.
-    int64 cardinality = 3;  /// Cardinality of the parameters.
+    string appId = 1;       /// The ID of the app the request was executed in.
+    string userId = 2;      /// The ID of the user the request was executed by.
+    int64 cardinality = 3;  /// The cardinality of the parameters.
 }
 
 /**
- * The communication service provited between Qlik Engine and the plugin.
+ * A header sent before returning data to Qlik, under the key "qlik-tabledescription-bin".
+ */
+ message TableDescription {
+    repeated FieldDescription fields = 1;     /// The fields of the table.
+    string name = 2;                          /// The name of the table.
+    int64 numberOfRows = 3;                   /// Number of rows in table.
```

Comment:
To be consistent with the rest of the comments, you might want to use "The number of rows in the table." instead.

Reply:
Good catch, this change must be updated on master and in qlik-oss/server-side-extentions as well, if we choose to correct it.

Reply:
No we wait with this change since it is too small and does not justify the work. We change it in future versions.